pyproject.toml

```diff
@@ -4,9 +4,8 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "chtools"
```

Comment:
it doesn't relate to this PR but I propose to rename it to `ch-tools`.

It should be possible to resolve issue with import name with the following directive:
```
[tool.flit.module]
name = "chtools"
```
https://flit.pypa.io/en/latest/pyproject_toml.html#module-section

Reply:
I'll update this once all pipelines are working.